code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -33,126 +37,174 @@ LINEN BINS
+	nightmare_messages = list("the clown", "a toolbox", "sHiTcUrItY", "why did you put them in for 50 minutes")
+
 
 /obj/item/weapon/bedsheet/hop
 	name = "head of personnel's bedsheet"
 	desc = "It is decorated with a key emblem.  For those rare moments when you can rest and cuddle with Ian without someone screaming for you over the radio."
 	icon_state = "sheethop"
 	item_color = "hop"
+	dream_messages = list("authority", "a silvery ID", "obligation", "a computer", "an ID", "a corgi", "the head of personnel")
+	nightmare_messages = list("improper paperwork", "all access clown", "50 open clown slots", "dead ian")
 
 /obj/item/weapon/bedsheet/ce
 	name = "chief engineer's bedsheet"
 	desc = "It is decorated with a wrench emblem.  It's highly reflective and stain resistant, so you don't need to worry about ruining it with oil."
 	icon_state = "sheetce"
 	item_color = "chief"
+	dream_messages = list("authority", "a silvery ID", "the engine", "power tools", "an APC", "a parrot", "the chief engineer")
+	nightmare_messages = list("forced airlock", "syndicate bomb", "explosion in research chem", "iT's LoOsE")
 
 /obj/item/weapon/bedsheet/qm
 	name = "quartermaster's bedsheet"
 	desc = "It is decorated with a crate emblem in silver lining.  It's rather tough, and just the thing to lie on after a hard day of pushing paper."
 	icon_state = "sheetqm"
 	item_color = "qm"
+	dream_messages = list("a grey ID", "a shuttle", "a crate", "a sloth", "the quartermaster")
+	nightmare_messages = list("a blad person", "no points", "wheres the ore", "space carp")
 
 /obj/item/weapon/bedsheet/brown
 	icon_state = "sheetbrown"
 	item_color = "cargo"
+	dream_messages = list("brown")
+	nightmare_messages = list("dead monkey")
 
 /obj/item/weapon/bedsheet/centcom
 	name = "centcom bedsheet"
 	desc = "Woven with advanced nanothread for warmth as well as being very decorated, essential for all officials."
 	icon_state = "sheetcentcom"
 	item_color = "centcom"
+	dream_messages = list("a unique ID", "authority", "artillery", "an ending")
+	nightmare_messages = list("a butt fax")
 
 /obj/item/weapon/bedsheet/syndie
 	name = "syndicate bedsheet"
 	desc = "It has a syndicate emblem and it has an aura of evil."
 	icon_state = "sheetsyndie"
 	item_color = "syndie"
+	dream_messages = list("a green disc", "a red crystal", "a glowing blade", "a wire-covered ID")
+	nightmare_messages = list("stunbaton", "taser", "lasers", "a toolbox")
 
 /obj/item/weapon/bedsheet/cult
 	name = "cultist's bedsheet"
 	desc = "You might dream of Nar'Sie if you sleep with this.  It seems rather tattered and glows of an eldritch presence."
 	icon_state = "sheetcult"
 	item_color = "cult"
+	dream_messages = list("a tome", "a floating red crystal", "a glowing sword", "a bloody symbol", "a massive humanoid figure")
+	nightmare_messages = list("a tome", "a floating red crystal", "a glowing sword", "a bloody symbol", "a massive humanoid figure")

```

Comment:
copy paste issue? why the same?

Reply:
cause its a cult..the dreams are the same as a nightmare...

Reply:
I mean, to a cultist, they're unlikely to be. More like `holy water` and shit.

Reply:
>You might dream of Nar'Sie

>You don't dream of Nar'Sie

Reply:
no. I had a reason.

Reply:
Cool, maybe write the reason? Because you haven't given a legitimate one yet.

Reply:
When you are being mind controlled by an eldritch abomination, your mind no longer works normally. Dreams and nightmares blend into a single dream state from which your waking moments offer no respite, as the reality of your life may seem more surreal to your shattering mind.

Or something.

Reply:
Well why not just make it so cult can only dream? Nightmares seem kind of oxy-moronic by that point

Reply:
Cult bedsheets are tainted with Narnar essence, it doesn't matter if you are dreaming, you're still going to have a nightmare. There's no way to differentiate between a cult member and a regular person in this code, nor should there be, because it would be pointless for 99% of bedsheets.